apps/ui/src/composables/useDelegates.ts

```diff
@@ -73,6 +74,49 @@ const DELEGATES_QUERY = gql`
   }
 `;
 
+const DELEGATES_QUERY_COMPOUND_GOVERNOR = gql`
+  query (
+    $first: Int!
+    $skip: Int!
+    $orderBy: Delegate_orderBy!
+    $orderDirection: OrderDirection!
+  ) {
+    delegates(
+      first: $first
+      skip: $skip
+      orderBy: $orderBy
+      orderDirection: $orderDirection
+      where: { tokenHoldersRepresentedAmount_gte: 0 }
+    ) {
+      id
+      delegatedVotes
+      delegatedVotesRaw
+      tokenHoldersRepresentedAmount
+    }
+    governance(id: "GOVERNANCE") {
+      delegatedVotes
+      totalDelegates
+    }
+  }
+`;
+
+const DELEGATIONS_PARAMS: Partial<
+  Record<DelegationType, { userField: string; query: DocumentNode }>
+> = {
+  'compound-governor': {
+    userField: 'id',
+    query: DELEGATES_QUERY_COMPOUND_GOVERNOR
+  },
+  'governor-subgraph': {
+    userField: 'user',
+    query: DELEGATES_QUERY_GOVERNOR_SUBGRAPH
```

Comment:
May need a better name for `DELEGATES_QUERY_GOVERNOR_SUBGRAPH` and `DELEGATES_QUERY_COMPOUND_GOVERNOR`